iOS/Flows/Conversation/Conversation Header/MembersViewController.swift

```diff
@@ -39,28 +42,37 @@ class MembersViewController: DiffableCollectionViewController<MembersCollectionV
         ConversationsManager.shared.$activeConversation
             .removeDuplicates()
             .mainSink { [unowned self] conversation in
-                Task {
-                    guard let cid = conversation?.cid else {
-                        await self.dataSource.deleteAllItems()
+                self.loadDataTask?.cancel()
+
+                if let cid = conversation?.cid {
+                    self.conversationController = ConversationController.controller(cid)
+                } else {
+                    self.conversationController = nil
+                }
```

Comment:
First we stop any member updates that were happening before. Then immediately set our conversation controller so that other functions can use it immediately.